src/mlpack/third_party/CLI/CLI11.hpp

```diff
@@ -0,0 +1,8789 @@
+#pragma once
```

Comment:
We probably should put this in `src/mlpack/bindings/cli/third_party/` (or something like that, just under the `src/mlpack/bindings/cli/` directory), since that's the only place that it's actually used.

Reply:
I will move it there immediately